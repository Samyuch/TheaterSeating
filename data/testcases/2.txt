  2 4 2
3 6 6 6 3
4 8 8 8 4
4 8 8 8 4
3 6 6 6 3
  2 4 2

James 4
Michale 10
Richard 8
Parker 100
Tyson 3
Williams 4
Brown 8
Miller 12
Max 0
Tate 6
Emily 7
Kenny 5
Tala 98