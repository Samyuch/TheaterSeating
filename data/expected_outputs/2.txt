James - Row 1 Section 2
Michale - Call to split party.
Richard - Row 3 Section 2
Parker - Call to split party.
Tyson - Row 2 Section 1
Williams - Row 4 Section 1
Brown - Row 3 Section 3
Miller - Call to split party.
Max - Invalid request. Either name and/or requested seats have invalid entry.
Tate - Row 2 Section 2
Emily - Row 3 Section 4
Kenny - Row 2 Section 3
Tala - Sorry, we can't handle your party.